src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/StandardRequestManager.java

```diff
@@ -258,9 +238,9 @@ public IRequest getRequestForToken(@NotNull final IToken<?> token)
             return null;
         }
 
-        final NBTTagCompound requestData = getFactoryController().serialize(internalRequest);
+        //final NBTTagCompound requestData = getFactoryController().serialize(internalRequest);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)